src/app/donation-start/donation-start-form/donation-start-form.component.html

```diff
@@ -401,22 +401,22 @@ <h3 class="b-rt-0 b-m-0 b-bold"><span class="span-hr"></span>How does Big Give u
                     }
                   </div>
                 </div>
-                <div>
-                  <biggive-form-field-select
-                    [prompt]="'Billing country'"
-                    [options]="countryOptionsObject"
-                    [selectedValue]="selectedCountryCode"
-                    [backgroundColour]="'grey'"
-                    [selectionChanged]="setSelectedCountry"
-                    [spaceBelow]="3"
-                    [selectedOptionColour]="'inherit'"
-                    >
-                  </biggive-form-field-select>
+                <div [style.display]="isSavedPaymentMethodSelected ? 'none' : 'block'">
```

Comment:
yesterday this was within a div with the stripe payment element that was hidden if the donor selected a saved card (or had donation funds)